### PR TITLE
Typos, spelling, and grammar corrected in documentation

### PR DESCRIPTION
**Pull request by linagee**
Thanks to aspell and languagetool. Hand picked from these, there are too
many strange things in the English language to be perfectly syntactical.

Linked to:

- https://github.com/meshtastic/Meshtastic-python/pull/181
- https://github.com/meshtastic/Meshtastic-protobufs/pull/62
- https://github.com/meshtastic/Meshtastic-device/pull/1031

### CODE REVIEW

**Comment by mkinney** [comment]
impressive!

File: 1.1 1.5 feature list ideas.md [hunk 1]
```diff
@@ -32,23 +32,23 @@ Probably later:
...
 * add text-to-speech option in the app, for hands/screen free monitoring (while busy with stuff) and warnings of system events (new node discovered, low battery etc...)
 * add speech-to-text to the app, now it can be used almost as a normal phone.
-* External buttons - Have firmware recognise several exposed GPIO pins for 'scroll', 'event', etc. so that external mom.spst (reed sw, mercury sw. etc) can trigger these events. At present, we must expose device to the elements, or manufacture weatherproof actuators (difficult).
-* Let app distribute APKs of itself over bluetooth or wifi: per [this forum post](https://meshtastic.discourse.group/t/suggestion-ability-to-transfer-app-apk-to-another-android-device-via-bt-or-other-non-network-dependent-method/711). let the device distribute the app or have a wifi web client for basic use for any wifi enabled device.
+* External buttons - Have firmware recognize several exposed GPIO pins for 'scroll', 'event', etc. so that external mom.spst (reed sw, mercury sw. etc) can trigger these events. At present, we must expose device to the elements, or manufacture weatherproof actuators (difficult).
```
Comment (Vertikar) [inline 1.1 1.5 feature list ideas.md:43]: You changed it to the wrong spelling! 😄

Comment (sachaw) [reply 1.1 1.5 feature list ideas.md:43]: These two markdown files in the repo root are going to get removed, they were placed here temporarily so we could clean up the device repo.

Comment (Vertikar) [reply 1.1 1.5 feature list ideas.md:43]: Was just poking fun as it was changed it from English to American English spellling 😄

**Pull request merged**